src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -63,13 +63,14 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     val dir = getConfiguredDirectory("deposits.stage-zips")
     logger.info(s"Uploads are staged in $dir")
     if (dir.nonEmpty) {
-      val msg = s"Pending uploads were probably interrupted. See logs lines with 'POST /deposit/{id}/file/{dir_path}'. Please remove their directories from the staging area: $dir"
+      val msg = s"Possibly pending or interrupted uploads the in staging area: $dir. See log lines with 'POST /deposit/{id}/file/{dir_path}'."
       logger.error(msg)
```

Comment:
Should you still do `logger.error` here? Given that you throw an exception here as well, doesn't this crash the application and write that to the `*.err` log file already? No this error log probably ends up in two places.